ballot/admin/referendum.py

```diff
@@ -2,4 +2,10 @@
 
 from ..models import referendum as models
 
-admin.site.register(models.Referendum)
+
+class ReferendumAdmin(admin.ModelAdmin):
+    fields = ('name', 'number', 'ballot',
+              'website_url', 'facebook_url', 'twitter_url')
```

Comment:
Can you add comments to each of these, as to why you're doing this? 

A big major downside of listing fields explicitly like like this is, if new fields are added to the model, then they must also be added here (which isn't true for most models).

Here and elsewhere, let's add an `assert` (or a test) that will notify us if things change.
`assert len(fields) - len(Referendum._meta.get_fields()) == some_num, "Make sure there are no new fields."